media_driver/linux/common/os/i915/include/mos_bufmgr.h

```diff
@@ -239,6 +239,21 @@ int mos_get_aperture_sizes(int fd, size_t *mappable, size_t *total);
 int mos_bufmgr_gem_get_devid(struct mos_bufmgr *bufmgr);
 
 struct mos_linux_context *mos_gem_context_create(struct mos_bufmgr *bufmgr);
+struct mos_linux_context *mos_gem_context_create_v2(struct mos_bufmgr *bufmgr, __u32 flags);
+struct class_instance {
+	uint16_t engine_class;
+	uint16_t instance;
+};
+#define MAX_VDBOX_NUM 64
```

Comment:
why 64? where this comes from? How many vdbox engines does i915 uAPI implies? or they get a count from us somehow for the buffers to fill?

Reply:
yes, i915 gets a buffer from UMD to fill when UMD queries engine instances. MAX_VDBOX_NUM is used to define the buffer size when we query the VDBOX engine instances.